guideline.md

```diff
@@ -0,0 +1,126 @@
+# Pyladies Germany PySV Grant Program
+
+The annual €5000 grant program from the PySV for PyLadies aims to support the activities and growth of PyLadies chapters in Germany. It is essential to ensure that these funds are used transparently, therefore we have devised the following guidelines. The grant will be issued to PyLadies each year after the PyCon DE conference (circa June), once assigned the funds are available for disposal by the PyLadies using the following guidelines. Money not used within the year will stay on the PyLadies account and can continue to be used.
+
+**Allowable expenses:**
+- Community events: this includes room rentals, refreshments, and similar costs.
+- Promotional materials: such as stickers, banners, T-shirts, and other marketing materials.
+- Software tools: Subscription-based tools or one-time software purchases that benefit the chapter (these may also be shared across chapters).
+- To pay/reimburse costs to speakers for PyLadies events
+- Travel reimbursement for members to go to other events (note the regional restrictions)
+
+**Disallowed Expenditures:**
+- Regional Restrictions: Funds can only be spent on activities in the DE, CH, and AT regions as defined in the PySV Bylaws.
+- Personal expenses or any activity not related to PyLadies.
+- It should be avoided that the opportunities funded benefit repeatedly only a single individual or small group of the chapter members within a single year
+
+## Accessing funds
+
+Chapters wishing to apply for funds should:
+- Open a ticket on our platform (to be set up) with a brief statement of the intended use of the funds, detailing the purpose and amount requested.
+- If necessary wait for approval (see process below)
```

Comment:
should we estipulate approving time? to better manage expectations

Reply:
agree, I am just not sure what is realistic for this. I have set up a Slack integration so we would get messaged about new requests and my gut feeling is fair is 2-3 business (week) days but I am not sure. What do you think @laysauchoa